js-miniapp-bridge/src/types/fetch.ts

```diff
@@ -0,0 +1,111 @@
+type Headers = Record<string, string>;
+
+type ResponseType = 'cors' | 'error';
+interface ResponseBody {
+  readonly body: Uint8Array | null;
+  readonly bodyUsed: boolean;
+  /**
+   * Decode response as ArrayBuffer
+   */
+  arrayBuffer(): Promise<ArrayBuffer>;
+  /**
+   * Decode response as json
+   */
+  json(): Promise<object>;
+  /**
+   * Decode response as text
+   */
+  text(): Promise<string>;
+}
+
+export class FetchResponseImpl implements Response {
```

Comment:
would it be okay if we keep this implementations of bridge-types in js-SDK workspace?

Reply:
I don't think that the `sdk` workspace is the right place for this. The `js-miniapp-sdk` npm module is mainly intended as a wrapper for the bridge functionality, so we try not to put much in there beyond what we have to.

Also, keeping this in the `bridge` workspace couples it with the native side because the bridge JS gets bundled with the native SDK. So if we make changes to the native side then we can simultaneously makes changes to the bridge JS. This is important in case we need to make changes in the bridge which would otherwise be breaking for mini apps that included an old version of `js-miniapp-sdk`.

Reply:
👍 Got it. Thanks for clarifying